packages/orbit-components/src/Stepper/README.md

```diff
@@ -56,32 +59,34 @@ import StepperStateless from "@kiwicom/orbit-components/lib/Stepper/StepperState
-| ariaLabelledBy    | `string`                    |         | Optional prop for `aria-labelledby` value.              |
+| Name                 | Type                        | Default | Description                                                                               |
+| :------------------- | :-------------------------- | :------ | :---------------------------------------------------------------------------------------- |
+| dataTest             | `string`                    |         | Optional prop for testing purposes.                                                       |
+| disabled             | `boolean`                   | `false` | If `true`, the Stepper will be disabled.                                                  |
+| disabledIncrement    | `boolean`                   |         | If `true`, the increment `Button` will be disabled.                                       |
+| disabledDecrement    | `boolean`                   |         | If `true`, the decrement `Button` will be disabled.                                       |
+| maxValue             | `number`                    | `∞`     | Specifies the maximum value for the Stepper.                                              |
+| minValue             | `number`                    | `-∞`    | Specifies the minimum value for the Stepper.                                              |
+| name                 | `string`                    |         | The name for the Stepper.                                                                 |
+| onBlur               | `event => void \| Promise`  |         | Function for handling onBlur event.                                                       |
+| onChange             | `number => void \| Promise` |         | Function for handling onClick event.                                                      |
+| onDecrement          | `event => void \| Promise`  |         | Function for handling decrement event.                                                    |
+| onFocus              | `event => void \| Promise`  |         | Function for handling onFocus event.                                                      |
+| onIncrement          | `event => void \| Promise`  |         | Function for handling increment event.                                                    |
+| onKeyDown            | `event => void \| Promise`  |         | Function for handling onKeyDown event present on input.                                   |
+| step                 | `number`                    | `1`     | Specifies the value of step to increment and decrement.                                   |
+| titleDecrement       | `string \| (any => string)` |         | Specifies `aria-label` property on decrement `Button`. See accessibility tab.             |
+| titleIncrement       | `string \| (any => string)` |         | Specifies `aria-label` property on increment `Button`. See accessibility tab.             |
```

Comment:
It looks like type of these is just string now, can you update it, please? Same for Stepper props table.